LondonBritishLibrary/orient/BLorient786.xml

```diff
@@ -0,0 +1,250 @@
+<textLang mainLang="gez"></textLang>
+<incipit xml:lang="gez"><locus target="#72v"></locus>በከመ፡ ተሠገወ፡ ቃለ፡ እግዚአብሔር፡ እመንፈስ፡ ቅዱስ፡ ወእማርያም፡</incipit>
+</msItem>
+
+<msItem xml:id="ms_i1.10">
+<locus from="88v"/>
+<title type="complete" xml:lang="en" ref="LIT6048OnPalmSundayRetuaH">For Palm Sunday</title>
+<textLang mainLang="gez"></textLang>
+<incipit xml:lang="gez"><locus target="#88v"></locus>ዘይትነበብ፡ በዕለተ፡ ሆሣዕና፡</incipit>
+</msItem>
+
+<msItem xml:id="ms_i1.11">
+<locus from="92v"/>
+<title type="complete" xml:lang="en" ref="LIT2149OntheW">On our Lord washing the feet of his disciples, and on the Last Supper</title>
+<textLang mainLang="gez"></textLang>
+<incipit xml:lang="gez"><locus target="#92v"></locus>በእንተ፡ ዘከመ፡ ሐፀበ፡ እግረ፡ አርዳኢሁ፡ ወዘከመ፡ ወሀበ፡ ኅብስተ፡ ወወይነ፡ እምድኅረ፡ ድራር፡ ለአርዳኢሁ፡</incipit>
+</msItem>
+
+<msItem xml:id="ms_i1.12">
+<locus from="109v"/>
+<title type="complete" xml:lang="en" ref="LIT2118OntheD">On the death of our Lord</title>
+<textLang mainLang="gez"></textLang>
+<incipit xml:lang="gez"><locus target="#109v"></locus>በእንተ፡ ሞቱ፡ ለእግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ ጊዜ፡ ፱ሰዓት፡ በዲበ፡ ዕፀ፡ መስቀል፡
+ወበእንተ፡ ፈያታዊ፡ ዘየማን፡ ወበእንተ፡ ሞት፡ ወሰይጣን፡ ዘከመ፡ ተበሃሉ፡</incipit>
+</msItem>
+
+<msItem xml:id="ms_i1.13">
+<locus from="125r"/>
+<title type="complete" xml:lang="en" ref="LIT2109OntheA">On the ascension of our Lord</title>
+<textLang mainLang="gez" xml:lang="gez"></textLang>
+<incipit xml:lang="gez"><locus target="#125r"></locus>ድርሳን፡ በእንተ፡ ዘከመ፡ ዓርገ፡ እግዚእነ፡ ውስተ፡ ሰማያት፡ መልዕልተ፡ ኵሉ፡ ስም፡ ወነበረ፡ በየማነ፡ ኃይል፡</incipit>
+</msItem>
+
+<msItem xml:id="ms_i1.14">
+<locus from="132v"/>
+<title type="complete" xml:lang="en" ref="LIT2137OntheP">On the Trinity in unity</title>
+<textLang mainLang="gez"></textLang>
+<incipit xml:lang="gez"><locus target="#132v"></locus>ርድአኒ፡ ወልደ፡ እግዚአብሔር፡ ከመ፡ እትናገር፡ በእንተ፡ ህሉና፡ መስልሥትከ፡ በተዋህዶትከ፡ </incipit>
+</msItem>
+
+<msItem xml:id="ms_i1.15">
+<locus from="148v"/>
+<title type="complete" xml:lang="en" ref="LIT6047On12ApostlesRetuaH">On the twelve apostles</title>
+<textLang mainLang="gez">በእንተ፡ ፲ወ፪ሐዋርያት፡ ዘይትነበብ፡ አመ፡ ፭ሐምሌ</textLang>
+<incipit xml:lang="gez"><locus target="#148v"></locus>ርድአኒ፡ ወልደ፡ እግዚአብሔር፡ ከመ፡ እትናገር፡ በእንተ፡ ህሉና፡ መስልሥትከ፡ በተዋህዶትከ፡ </incipit>
+</msItem>
+
+<msItem xml:id="ms_i1.16">
+<locus from="154r"/>
+<title type="complete" xml:lang="en" ref="LIT2145OntheT">On the transfiguration of our Lord</title>
+<textLang mainLang="gez"></textLang>
+<incipit xml:lang="gez"><locus target="#154r"></locus>ድርሳን፡ ዘደብረ፡ ታቦር፡ ዘከመ፡ ተወለጠ፡ ራዕዩ፡ ለእግዚእነ፡ በውስተ፡ ደብረ፡ ታቦር፡</incipit>
+</msItem>
+
+<msItem xml:id="ms_i1.17">
+<locus from="160v"/>
+<title type="complete" xml:lang="en" ref="LIT2124OntheF">On the four beasts</title>
+<textLang mainLang="gez"></textLang>
+<incipit xml:lang="gez"><locus target="#160v"></locus>ድርሳን፡ ዘ፬እንስሳ፡ ክቡራን፡</incipit>
+</msItem>
+
+
+
+<colophon xml:id="coloph1" xml:lang="gez">
+        <locus from="167v"/>
+        ዝመጽሐፍ፡ ዘእራስ፡ <persName role="owner" ref="PRS2021IHA">ሣህለ፡ ሥላሴ፡</persName>
```

Comment:
it would be better to use rather PRS8346sahlase (they are doubblets, but once there is an ID of BM and not of IHA, we prefer that of BM). It is very confusing, I know